org-clubhouse.el

```diff
@@ -663,11 +663,11 @@ If the epics already have a CLUBHOUSE-EPIC-ID, they are filtered and ignored."
                         (org-make-link-string
                          (org-clubhouse-link-to-story story-id)
                          (number-to-string story-id)))
-
-      (org-set-property "clubhouse-epic"
-                        (org-make-link-string
-                         (org-clubhouse-link-to-epic epic-id)
-                         (alist-get epic-id (org-clubhouse-epics))))
+ (when epic-id
```

Comment:
indentation looks off here

Reply:
Yep, as stated by https://github.com/glittershark/org-clubhouse/pull/24#discussion_r368311922

My indentation config is whack. Which mode do you use/configured of elisp?

Reply:
I just go with whatever `lisp-indent-line` in emacs does automatically.

Reply:
Mistery solved! Thanks! :)